home/tests/test_whatsapp.py

```diff
@@ -166,10 +187,47 @@ def test_create_whatsapp_template_with_image(self, tmp_path, settings):
             ],
         }
 
-        assert (
-            create_template_request.headers["Authorization"]
-            == "Bearer fake-access-token"
-        )
-        assert create_template_request.body == json.dumps(
-            mock_create_template_data, indent=4
-        )
+        assert ct_req.headers["Authorization"] == "Bearer fake-access-token"
+        assert json.loads(ct_req.body) == mock_create_template_data
+
+    @responses.activate
+    def test_create_whatsapp_template_with_language(self) -> None:
+        """
+        When we create a WhatsApp template, the language is converted to the
+        appropriate WhatsApp language code.
+        """
+        url = "http://whatsapp/graph/v14.0/27121231234/message_templates"
+        responses.add(responses.POST, url, json={})
+
+        # The default English locale gives us a language of "en_US".
+        # FIXME: Should this be "en" instead?
```

Comment:
It looks like we can define both in our settings file: https://docs.wagtail.org/en/stable/reference/settings.html#wagtail-content-languages

Maybe we should ensure that that list matches what we can do with WhatsApp templates? We probably don't want to remove any languages from the list, but we can do something like have all 3 of `en`, `en-us`, `en-gb`, and then for backwards compatibility add a migration that changes any `en` locales to `en-us`

Reply:
This (somewhat poorly worded, sorry) FIXME was about switching the default WhatsApp language for wagtail's `en` from the current `en_US` to `en`. I think we've already decided to keep that `en_US` for backwards compatibility with existing templates.